loc/en.js

```diff
@@ -23,8 +23,19 @@ module.exports = {
     scan: 'Scan',
     storage_is_encrypted: 'Your storage is encrypted. Password is required to decrypt it',
   },
+  time: {
+    days: 'Days',
+    hours: 'Hours',
+    minutes: 'Minutes',
+  },
   airdrop: {
     title: 'Airdrop',
+    dateOfAirdrop: 'Date of Airdrop: ',
```

Comment:
Do we want to have spaces like this in translations for sure?

Reply:
done